apps/service_providers/utils.py

```diff
@@ -139,3 +140,19 @@ def get_llm_provider_choices(team) -> dict[int, dict[str, list[dict[str, Any]]]]
             "supports_assistants": provider.type_enum.supports_assistants,
         }
     return providers
+
+
+def get_first_llm_provider():
+    try:
+        return LlmProvider.objects.order_by("id").first()
+    except ObjectDoesNotExist:
```

Comment:
The Django way to do this is:
```suggestion
    except LlmProvider.DoesNotExist:
```